### PR TITLE
feat: allow configuring many hosts at once

### PR DESCRIPTION
**Pull request by omrozowicz-splunk**
# Description

This PR introduces a method to configure many hosts at once in inventory

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist

- [x] My commit message is [conventional](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] I have run pre-commit on all files before creating the PR
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: integration_tests/test_poller_integration.py [hunk 1]
```diff
@@ -437,6 +438,89 @@ def test_snmpv3_walk(self, setup_splunk):
...
+            "varBinds": [yaml_escape_list(sq("IP-MIB"))],
+        },
+        "switches_profile": {
+            "frequency": 7,
+            "varBinds": [yaml_escape_list(sq("IP-MIB"))],
+        },
+    }
+    groups = {
+        "routers": [{'address': trap_external_ip, 'port': 1163}, {'address': trap_external_ip, 'port': 1164}],
+        "switches": [{'address': trap_external_ip}, {'address': 'trap_external_ip', 'port': 1162}],
```
Comment (wojtekzyla) [inline integration_tests/test_poller_integration.py]: should 'trap_external_ip' be a string instead of variable?

File: test-results/junit.xml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+    def _dot_lookup(thing, comp, import_path):
+        try:
+&gt;           return getattr(thing, comp)
+E           AttributeError: module 'splunk_connect_for_snmp.snmp' has no attribute 'tasks'
```
Comment (omrozowicz-splunk) [inline test-results/junit.xml:6]: I think we should delete this file

File: poetry.lock [hunk 3]
```diff
@@ -27,25 +27,25 @@ python-versions = ">=3.6"
 
 [[package]]
 name = "atomicwrites"
-version = "1.4.0"
+version = "1.4.1"
```
Comment (omrozowicz-splunk) [inline poetry.lock:30]: let's not update poetry.lock this way

**Pull request merged**